lib/feedback.js

```diff
@@ -21,7 +21,7 @@ var debug = require('debug')('lib/feedback')
 var _ = require('lodash')
 var shortid = require('shortid')
 
-function feedback(system) {
+function feedbacks(system) {
```

Comment:
Don't know that I agree with changing the class name while the file name stays the same ... not a huge deal.  Was there a scope or variable naming reason for this change?

Reply:
This was a mistake. I changed it, and then attempted to undo that but made a typo

Reply:
fixed